src/test/resources/org/zalando/nakadi/validation/strict-validation.json

```diff
@@ -0,0 +1,261 @@
+[
```

Comment:
Lot's of tests for checking strict validation. It illustrates all scenarios I could think of when we are able to enforce strict validation.